posydon/popsyn/defaults.py

```diff
@@ -15,10 +15,11 @@
 
     # Size of the population
     'number_of_binaries': 100,
-    #Eirini's changes 
+    #Eirini's changes
```

Comment:
Avid this type of comment, the repository is now public. With `git blame` or the git extension for VS code, you can see directly who is the author of each line of code.